js/sponsor.js

```diff
@@ -0,0 +1,47 @@
+// "<div class='row text-center'><h3>Platinum Sponsors</h3><div class='col-md-4'>\
+// <img src='img/border2.png' class='border2 border2a' alt='sponsor-border'>\
+// <img src='img/micro.png' alt='sponsor-image' class='img-responsive sponsora center-block'>\
+// </div></div>";
+$(document).ready(()=>{
+
+    $.ajax({
+        url:'./assets/sponsors.json',
+        beforeSend: (xhr)=>{
+            xhr.setRequestHeader("Access-Control-Allow-Origin","*");
+        }  
+    }).done((response,status,xhr)=>{
+        if(match(status,"success")){
+            findandseparate(response);
+        }
+        console.log("status not OK",status);
+    }).fail((err)=>{
+        console.log("Error",err);
+    }).always(()=>{
+
+    });
+});
+
+match = function(str1,str2){
+    return str1 === str2;
+}
+
+findandseparate = function(response){
```

Comment:
Use a `var` here as well

Reply:
Also rename it to something good like `constructSponsorBlock` . Also always use camel casing for variable names.